src/blog/components/informationMessage/InformationMessage.tsx

```diff
@@ -0,0 +1,67 @@
+import Button from "src/components/buttons/Button";
+import LinkButton from "src/components/linkButton/LinkButton";
+import Text from "src/components/text/Text";
+import { ILink, LinkType } from "studio/lib/payloads/navigation";
+import styles from "./informationMessage.module.css";
+import React from "react";
+
+export const homeLink = {
+  _key: "return-home",
+  _type: "link",
+  linkTitle: "Return to home",
+  linkType: LinkType.Internal,
+  internalLink: {
+    _ref: "/",
+  },
+};
+
+export const studioLink = {
+  _key: "go-to-studio",
+  _type: "link",
+  linkTitle: "Go to studio",
+  linkType: LinkType.Internal,
+  internalLink: { _ref: "/studio" },
+};
+
+interface InformationMessageProps {
+  title: string;
+  body: string;
+  button?: {
+    label: string;
+    onClick: () => void;
+  };
+  link?: ILink;
+}
+
+const InformationMessage = ({
+  title,
+  body,
+  button,
+  link,
+}: InformationMessageProps) => {
+  return (
+    <section className={styles.wrapper} role="status" aria-live="assertive">
+      <div className={styles.info}>
+        <Text type="h1">{title}</Text>
+        <span>
+          {body.split("\n").map((line, index) => (
```

Comment:
is this possible to to in page.tsx? because now every single place we use InformationMessage it'll split the sentences and add a line-break

Reply:
It will only split on "\n", so to get linespace, the string typed body needs to containt \n